### PR TITLE
feat: split new flow for policy violations - Part 4

### PR DESCRIPTION
**Pull request by suyashpatil78**
### Description
copilot:summary

copilot:poem

### Walkthrough
copilot:walkthrough

## Clickup
app.clickup.com

## Code Coverage
Please add code coverage here

## UI Preview
Please add screenshots for UI changes

### CODE REVIEW

File: src/app/fyle/split-expense/split-expense.page.ts [hunk 1]
```diff
@@ -709,7 +734,9 @@ export class SplitExpensePage {
             })
           )
           .subscribe((response) => {
-            console.log(response);
+            if (response && response.action === 'continue') {
```
Comment (Chethan-Fyle) [inline src/app/fyle/split-expense/split-expense.page.ts:737]: `if (response?.action === 'continue') {`

**Review by Chethan-Fyle — APPROVED** [review]

File: src/app/core/services/platform/v1/spender/expenses.service.ts [hunk 2]
```diff
@@ -218,6 +218,12 @@ export class ExpensesService {
     });
   }
 
+  splitExpense(params: SplitPayload) {
```
Comment (Dimple16) [inline src/app/core/services/platform/v1/spender/expenses.service.ts:221]: what is the return type?

**Review by Dimple16 — APPROVED** [review]
minor comment

File: src/app/core/services/platform/v1/spender/expenses.service.ts (hunk 2 above)
Comment (suyashpatil78) [reply src/app/core/services/platform/v1/spender/expenses.service.ts]: I have fixed the return types in eslint wali PRs, by the time I merge them, return types will be there

**Pull request merged**